src/pages/AddressPage/tabs/inventory/Inventory.tsx

```diff
@@ -20,6 +20,14 @@ export function Inventory(props: IInventoryProps) {
     (page + 1) * pageSize
   );
 
+  if (!!!inventory.length) {
+    return (
+      <Box style={{ height: "700px" }} justify="center" align="center">
+        <Spinner size="large" />
+      </Box>
+    );
+  }
+
```

Comment:
Do we need a set height here or do we have the availability of a helper class for padding here? I'd want to stay away from inline styling here if possible. I'm not familiar with grommet, but I'd rather see something that's built-in than a style prop being invoked here.